src/components/widgets/components/two-lines-text-banner/two-lines-text-banner.tsx

```diff
@@ -25,46 +25,55 @@ const TwoLinesStakeBannerComponent = (
     const data = module.data as I2LinesTextBannerData;
     const cta = module.cta as ICta;
 
-    return (
-        <TouchableOpacity
-            onPress={() => {
-                if (cta.type === 'navigateTo') {
-                    const blockchainConfig = getBlockchain(props.account.blockchain);
-
-                    const token: ITokenState =
-                        props.account.tokens[props.chainId][blockchainConfig.config.coin];
-
-                    NavigationService.navigate(cta.params.screen, {
-                        ...cta.params.params,
-                        blockchain: props.account.blockchain,
-                        accountIndex: props.account.index,
-                        token
-                    });
-                }
-            }}
-            activeOpacity={0.9}
+    const moduleJSX = (
+        <View
+            style={[
+                styles.container,
+                { backgroundColor: data?.backgroundColor || theme.colors.cardBackground },
+                module?.style && formatStyles(module.style)
+            ]}
         >
-            <View
-                style={[
-                    styles.container,
-                    { backgroundColor: data?.backgroundColor || theme.colors.cardBackground }
-                ]}
-            >
-                <View style={styles.textContainer}>
-                    <View style={styles.row}>
-                        {formatDataJSXElements(data.firstLine, styles.mainText)}
-                    </View>
-                    <View style={styles.row}>
-                        {formatDataJSXElements(data.secondLine, styles.secondaryText)}
-                    </View>
+            <View style={styles.textContainer}>
+                <View style={styles.row}>
+                    {formatDataJSXElements(data.firstLine, styles.mainText)}
+                </View>
+                <View style={styles.row}>
+                    {formatDataJSXElements(data.secondLine, styles.secondaryText)}
                 </View>
-
-                {data?.icon?.value && (
-                    <Icon name={data.icon.value} size={normalize(50)} style={styles.icon} />
-                )}
             </View>
-        </TouchableOpacity>
+
+            {data?.icon?.value && (
+                <Icon name={data.icon.value} size={normalize(50)} style={styles.icon} />
+            )}
+        </View>
     );
+
+    if (module?.cta) {
```

Comment:
for now it's fine, but we need a function to handle cta objects from API in a generic way. This is a generic component but cta functionality is limited